### PR TITLE
payment config and idam-pr fixes

### PR DESCRIPTION
**Pull request by smathangi**

### CODE REVIEW

File: ccd/values.yaml [hunk 1]
```diff
@@ -366,33 +364,34 @@ ccpay:
...
-      LIBERATA_KEYS_OAUTH2_CLIENT_ID:
-        disabled: true
-      LIBERATA_KEYS_OAUTH2_CLIENT_SECRET:
-        disabled: true
-      LIBERATA_KEYS_OAUTH2_USERNAME:
-        disabled: true
-      LIBERATA_KEYS_OAUTH2_PASSWORD:
-        disabled: true
+  payment-api:
+    enabled: true
```
Comment (mario-paniccia) [inline ccd/values.yaml]: I think this should be disabled by default

Comment (smathangi) [reply ccd/values.yaml]: whole pay product chart is disabled by default : https://github.com/hmcts/chart-ccd/blob/ccpay_fix/ccd/values.yaml#L35

Comment (mario-paniccia) [reply ccd/values.yaml]: 👍

File: ccd/values.yaml [hunk 2]
```diff
@@ -69,14 +69,14 @@ ccd-case-management-web:
       DM_URL_REMOTE: "^https?://(?:{{ tpl .Values.global.dmStoreUrl $}}(?::\\d+)?)/documents"
       ANNOTATION_API_URL: https://{{ tpl .Values.global.ccdApiGatewayIngress $}}/em-anno"
   idam-pr:
-    releaseNameOverride: ccd-idam-pr
+    releaseNameOverride: "{{ .Release.Name }}-ccd-idam-pr"
     api:
-      url: '{{ .Values.global.idamApiUrl }}'
+      url: '{{ tpl .Values.global.idamApiUrl }}'
```
Comment (timja) [inline ccd/values.yaml]:
```suggestion
      url: '{{ tpl .Values.global.idamApiUrl $ }}'
```

File: ccd/values.yaml [hunk 3]
```diff
@@ -69,14 +69,14 @@ ccd-case-management-web:
...
     web_public:
-      url: '{{ .Values.global.idamWebUrl }}'
+      url: '{{ tpl .Values.global.idamWebUrl }}'
```
Comment (timja) [inline ccd/values.yaml]:
```suggestion
      url: '{{ tpl .Values.global.idamWebUrl $ }}'
```

File: ccd/values.yaml [hunk 4]
```diff
@@ -69,14 +69,14 @@ ccd-case-management-web:
...
     redirect_uris:
       CCD:
-        - 'https://{{.Values.global.ccdCaseManagementWebIngress}}/oauth2redirect'
+        - 'https://{{ tpl .Values.global.ccdCaseManagementWebIngress}}/oauth2redirect'
```
Comment (timja) [inline ccd/values.yaml]:
```suggestion
        - 'https://{{ tpl .Values.global.ccdCaseManagementWebIngress $ }}/oauth2redirect'
```

**Pull request merged**